101-load-balancer-ip-configured-backend-pool/azuredeploy.json

```diff
@@ -343,7 +340,7 @@
             "name": "[concat(variables('lbName'), '/', variables('lbBackendPoolName'))]",
             "location": "[parameters('location')]",
             "dependsOn": [
-                "[resourceId('Microsoft.Network/loadBalancers', variables('lbName'))]",
+                "[resourceId('Microsoft.Network/loadBalancers', variables('lbName'))]"
             ],
             "properties": {
                 "loadBalancerBackendAddresses": [
```

Comment:
Ok, so if the intended behavior here is to just go 1, 2, 3 for the BE pool, we should use a copy loop here as we do for the others...

Reply:
So for this the intention is to create 1 BE pool with addresses 1, 2, and 3. I have tried to do a copy loop but it seems that this can only be done on a resource level and in this case it would need to be done on a property level otherwise it creates 3 separate BE pools

Reply:
replace "loadBalancerBackendAddresses" with this:

```
"copy": [
            {
                "name": "loadBalancerBackendAddresses",
                "count": "[length(variables('nicIpAddresses')]",
                "input": {
                    "name": "[concat('address', copyIndex('loadBalancerBackendAddresses'))]",
                    "properties": {
                        
                            "virtualNetwork": {
                                "id": "[resourceId('...)]"
                            },
                            "ipAddress": "[variables('nicIpAddresses')[copyIndex('loadBalancerBackendAddresses')]]"
                    }
                }
```